routes/chat.js

```diff
@@ -118,9 +80,45 @@ router.post('/chat', async function (req, res) {
       chat: chatID,
     });
 
-    res.json(chat);
+    res.status(201).json(chat);
   } else {
     // join
-    res.status(204).json(commonChats);
+    res.status(200).json(commonChats);
```

Comment:
@JekxDevil @Hwid2121 what does the server need to send to the client if the chat already exists?